mobile/src/main/java/de/michelinside/glucodatahandler/widget/GlucoseBaseWidget.kt

```diff
@@ -241,6 +245,37 @@ abstract class GlucoseBaseWidget(private val type: WidgetType,
             else
                 remoteViews.setViewVisibility(R.id.cobText, View.VISIBLE)
         }
+
+
+        if (hasBatteryLevel) {
+            var deviceNameStr = "No WearOS Device"
+            var levelStr = "?%"
+            var levelColour = Color.RED;
+
+            if (WearPhoneConnection.nodesConnected) {
+                if (WearPhoneConnection.connectionError) {
+                    deviceNameStr = "Connection error"
+                }
+                else {
+                    WearPhoneConnection.getNodeBatterLevels().firstNotNullOf { (name, level) ->
+                        if (level > 0)
+                            levelStr = "$level%"
+                        deviceNameStr = name
```

Comment:
Maybe we can simplify the name, while removing "(...)" For example "Galaxy Watch 5 (abc)" would be nicer to have only "Galaxy Watch 5".

Reply:
I only have my own watch as reference which doesn't have anything in braces after the name.. I can check for that case and remove if required.

Reply:
Here is my watch name
![Screenshot_20241223_163933_GlucoDataHandler](https://github.com/user-attachments/assets/9ab75973-9ece-47eb-9871-6f20aab1a31a)

Reply:
Or leave it as it is, I will change in the Wear class to have the same name everywhere.